a001.rb

```diff
@@ -0,0 +1,7 @@
+y = 0 
+(1...1000).each do |x|
+  if x %3 == 0 || x % 5 == 0
```

Comment:
I'm probably a little unusual, in that I would prefer this line to look like:
```
if (x % 3 == 0) || (x % 5 == 0)
```
It's true that order of precedence rules for operators makes the parens unnecessary.
It's also true that I am a human and find the explicit parentheses clearer & easier to mentally parse.
```